src/components/highlight/HighlightEntries.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import uuid from 'uuid/v1';
+import PropTypes from 'prop-types';
+
+import { HorizontalLine } from '../horizontalLine';
+import './styles.scss';
+
+const HighlightEntries = ({ highlights = [] }) => (
+  <div>
+    {!!highlights.length && <div><HorizontalLine />Highlighted Comments</div>}
```

Comment:
@Ibidapo consider changing line `10` to `{highlights.length > 0 && ... }`.

Reply:
Noted!